common/remote/sockfile.cpp

```diff
@@ -1964,7 +1966,20 @@ class CRemoteFile : public CRemoteBase, implements IFile
             Owned<CEndpointCS> crit = dirCSTable->getCrit(ep); // NB dirCSTable doesn't own, last reference will remove from table
             CriticalBlock block(*crit);
             sendBuffer.append((RemoteFileCommandType)RFCgetdir).append(dir).append(tail).append(includedirs).append(sub);
-            sendRemoteCommand(sendBuffer, replyBuffer);
+            try
+            {
+                sendRemoteCommand(sendBuffer, replyBuffer);
+            }
+            catch (IException * e)
```

Comment:
More accurate to catch IDAFS_Exception here.

Reply:
Changed